api/src/routes/channel/index.ts

```diff
@@ -11,14 +11,14 @@ export class ChannelRoutes {
 
 	createChannel = async (req: AuthenticatedRequest, res: Response, next: NextFunction): Promise<Response<any>> => {
 		try {
-			const { topics, seed, encrypted } = req.body as CreateChannelBody;
+			const { topics, seed, encrypted, hasPresharedKey, presharedKey, subscriptionPassword } = req.body as CreateChannelBody;
 			const { identityId } = req.user;
 
 			if (!identityId) {
 				return res.sendStatus(StatusCodes.BAD_REQUEST);
 			}
 
-			const channel = await this.channelService.create(identityId, topics, encrypted, seed);
+			const channel = await this.channelService.create(identityId, topics, encrypted, hasPresharedKey, seed, presharedKey, subscriptionPassword);
```

Comment:
To not accidentially mess up the order of parameters, I would suggest to send them as an object. 
In this case you will need to modify the `create()` function to also accept one object and take parameters from there.
```
const channel = await this.channelService.create({ identityId, topics, encrypted, hasPresharedKey, seed, presharedKey, subscriptionPassword });  // now parameters can be in any order
```